getsolus/packages
# msbuild: build fix & update to 16.9

**Summary**

Many internal changes between 16.8->16.9. Main reason for update is that it's the one mono 16.12.0.* expects.

Full list of changes [available here](https://github.com/dotnet/msbuild/blob/main/documentation/Changelog.md)

solves #2622 

**Test Plan**

 - build a few projects
 - build godot and confirm working

**Checklist**
